user-management/create_users.py

```diff
@@ -0,0 +1,42 @@
+import csv
+import fire
+import json
+import os
+import sys
+
+current_dir = os.path.dirname(os.path.realpath(__file__))
+parent_dir = os.path.dirname(current_dir)
+sys.path.append(parent_dir)
+
+from toolbox.get_access_token import get_access_token
+from toolbox.post_request import post_request
+
+
+def create_users(base_url, client_id, client_secret, input_file_path="./sample-files/create_users_sample.csv", email_verified=0):
+    try:
+        api_url = base_url + "/api/v1/users"
+        access_token = get_access_token(base_url, client_id, client_secret)
+
+        with open(input_file_path, newline='') as input_file:
+            users = csv.reader(input_file, delimiter=',')
+            for user in users:
+                new_user = {
+                    "email": user[0],
+                    "name": user[1],
+                    "password": user[2],
+                    "emailVerified": email_verified
+                }
+                response = post_request(api_url, access_token, new_user)
+
+                if "json" in response.headers["content-type"]:
```

Comment:
let's also check for status code

Reply:
ini mau dicek yg status codenya 200 aja kah?

Reply:
🤔 iya, aku pikirnya yang 200 print response.json

kalau bukan, raise error?

Reply:
selain raise Exception boleh tetep print response, tpi harus ada pembedanya gt dari yang sukses dan gagal